### PR TITLE
Add initial support for imx8qm board

### PR DESCRIPTION
**Pull request by unbel13ver**

### CODE REVIEW

**Comment by unbel13ver** [comment]
@vilvo

File: boards/nxp-imx8qm/default.nix [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        ref = "lf_v2022.04";
+    };
+
+    defconfig = "imx8qm_mek_defconfig";
+
+    useDefaultPatches = true;
+
+    builder = {
+      postPatch = ''
+        install -m 0755 $BL31/bl31.bin ./
```
Comment (vilvo) [inline boards/nxp-imx8qm/default.nix]: `0644` instead? Especially x-flag with copying all seems overly generous.

Comment (vilvo) [reply boards/nxp-imx8qm/default.nix]: The point here is that when we add binary blobs, let's not use executable flags when not absolutely required.

**Review by vilvo — CHANGES_REQUESTED** [review]
When we add binary blobs, let's not use executable flags when not absolutely required.

File: boards/nxp-imx8qm/default.nix (hunk 1 above)
Comment (unbel13ver) [reply boards/nxp-imx8qm/default.nix]: Thanks, for noticing this! Fixed.

**Review by vilvo — APPROVED** [review]

**Pull request merged**